gstd/src/async_runtime/locks.rs

```diff
@@ -181,6 +178,16 @@ impl LocksMap {
         locks.remove(&waiting_reply_to);
     }
 
+    pub fn remove_message_entry(&mut self, message_id: MessageId) {
+        self.0.remove(&message_id);
+        // Question for reviewers: how map for message can be not empty at this
+        // point ?!!! .map(|locks| {
+        //     if !locks.is_empty() {
+        //         // unreachable!("Wanna to remove locks map for message, but
+        // the map is not empty: {:?}", locks);     }
+        // });
```

Comment:
It can't so the panic on `None` removal res seems ok

Reply:
This panic is appeared some times, so we should discuss it more

Reply:
discussed